data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/unapproachable_east/una_spells.lst

```diff
@@ -78,6 +78,51 @@ Polymorph.MOD													CLASSES:Runescarred Berserker=5
 Alter Self.MOD													CLASSES:Runescarred Berserker=5
 Spell Resistance.MOD												CLASSES:Runescarred Berserker=5
 #
+# 1st level
+Cure Moderate Wounds.MOD	DESC:Cost of Runescarr %1 gp/%2 xp|5*1*classlevel("Runescarred Berserker")|5*1*classlevel("Runescarred Berserker")/25	PRECLASS:1,Runescarred Berserker=1
```

Comment:
Is there a reason why you want to take away these spells from all the other classes and to force all classes to see a 'Cost of Runescarr'?